app/src/main/java/org/gdg/frisbee/android/arrow/Organizer.java

```diff
@@ -0,0 +1,42 @@
+package org.gdg.frisbee.android.arrow;
+
+import com.google.android.gms.plus.model.people.Person;
+
+public class Organizer {
```

Comment:
I don't see another `Organizer` class in the project. Maybe I don't understand your point?